### PR TITLE
Prevent deleting current user

### PR DESCRIPTION
**Pull request by rn0**

### CODE REVIEW

File: Resources/views/Admin/datagrid.html.twig [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{% extends '@FSiAdmin/CRUD/datagrid.html.twig' %}
+
+{# https://github.com/fsi-open/datagrid-bundle/blob/06d45921087b47edc55d5d95555faf07e1ceec9f/Resources/views/datagrid.html.twig#L23 #}
+{# pattern: datagrid_{grid_name}_column_name_{column_name}_cell #}
+{% block datagrid_admin_security_user_column_name_batch_cell %}
+    <td>
+    {% if cell.value == app.user.id %}
+        <input type="checkbox" disabled />
```
Comment (szymach) [inline Resources/views/Admin/datagrid.html.twig]: nie lepiej w ogóle pominąć tworzenie checkboxa?

**Comment by rn0** [comment]
@fsi-open

File: EventListener/PreventDeletingCurrentUser.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        }
+    }
+
+    /**
+     * @param FormEvent $event
+     */
+    private function setRedirectResponse(FormEvent $event)
+    {
+        $event->stopPropagation();
+        $event->setResponse(new RedirectResponse($event->getRequest()->get('redirect_uri')));
```
Comment (JarekW) [inline EventListener/PreventDeletingCurrentUser.php]: when there's no `redirect_uri` parameter go to `UserElement` route

**Comment by rn0** [comment]
@fsi-open

**Comment by chives** [comment]
:+1:

**Comment by JarekW** [comment]
:+1:

**Pull request merged**